app/src/main/java/io/homeassistant/companion/android/launch/LaunchActivity.kt

```diff
@@ -62,4 +68,12 @@ class LaunchActivity : AppCompatActivity(), LaunchView {
     override fun getMenuInflater(): MenuInflater {
         return LokaliseMenuInflater(this)
     }
+
+    private fun startSensorWorker() {
+        val sensorWorker =
+            PeriodicWorkRequestBuilder<SensorWorker>(15, TimeUnit.MINUTES)
```

Comment:
Is this something we want to do periodically or only when:
 - we send a location event
 - we haven't send a location event in 30 min
 - we change charging type (non-charging -> charging)

Reply:
So in this initial implementation I was going for just every 15 ish minutes update all sensors.  I plan on listening for change events wherever possible to trigger an individual sensor to update.  I didn't want to link this functionality to the background location updates initially but there is nothing saying we can't.